frame/scored-pool/src/mock.rs

```diff
@@ -97,7 +97,9 @@ impl pallet_balances::Config for Test {
 }
 
 thread_local! {
-	pub static MEMBERS: RefCell<Vec<u64>> = RefCell::new(vec![]);
+
+	pub static MEMBERS: RefCell<BoundedVec<u64,ConstU32<10_u32>>> = RefCell::new(bounded_vec![0,10]);
+
```

Comment:
```suggestion
	pub static MEMBERS: RefCell<BoundedVec<u64,ConstU32<10_u32>>> = RefCell::new(bounded_vec![0,10]);
```

Reply:
Or better, ues: 

```
parameter_types! {
    pub static Members: BoundedVec<u64,ConstU32<10_u32>> = bounded_vec![0,10]
}
```

and then you get `Members::get()` and `Members::set()` with the same functionality.